--- entering/handwritten/mfe_plant/mfe_power_plant/cas23_to_28_capital_impl.py
+++ production/handwritten/mfe_plant/mfe_power_plant/cas23_to_28_capital_impl.py
@@ -2,7 +2,7 @@
 
 AUTO_IMPLEMENTED = True
 
-SysML Source: root-0/designs/generic_mfe/mfe_plant.sysml:570
+SysML Source: root-0/designs/generic_mfe/mfe_plant.sysml:942
 
 SysML Expressions:
 """
@@ -15,7 +15,7 @@
 def run_cas23_to_28_capital(inputs: cas23_to_28_capitalInput) -> float:
     """Execute cas23_to_28_capital calculation.
 
-SysML Source: root-0/designs/generic_mfe/mfe_plant.sysml:570
+SysML Source: root-0/designs/generic_mfe/mfe_plant.sysml:942
 
 Args:
     inputs: Input parameters validated against cas23_to_28_capitalInput schema
